spec/GraphQLOverHTTP.md

```diff
@@ -138,13 +156,15 @@ Note: {variables} and {extensions}, if set, must have a map as its value.
 
 ## GET
 
-For HTTP GET requests, the query parameters should be provided in the query component of the request URL in the form of
+For HTTP GET requests, the query parameters MUST be provided in the query component of the request URL in the form of
 `key=value` pairs with `&` symbol as a separator and both the key and value should have their "reserved" characters percent-encoded as specified in [section 2 of RFC3986](https://tools.ietf.org/html/rfc3986#section-2).
-The unencoded value of the {variables} parameter should be represented as a JSON-encoded string.
+The `variables` parameter should be represented as a URL-encoded JSON string.
```

Comment:
```suggestion
The `variables` parameter, if used, MUST be represented as a URL-encoded JSON string.
```

What is a server supposed to do with any other encoding, e.g. a URL-encoded XML string?